stopwatch.py

```diff
@@ -36,6 +36,9 @@
 import random as insecure_random
 import time
 
+TraceAnnotation = collections.namedtuple('TraceAnnotation', ['name', 'time'])
+TraceKeyValueAnnotation = collections.namedtuple('TraceKeyValueAnnotation', ['key', 'value'])
```

Comment:
I made this attempt
https://github.com/dropbox/stopwatch/pull/5

I like that approach better than this one, so I'd prefer to scrap this in favor of that.
I think we could fit that approach into the rest of the dropbox codebase pretty reasonably and I believe it's a better way to evolve stopwatch as a library.